aws/resource_aws_elasticache_replication_group.go

```diff
@@ -532,3 +569,16 @@ func validateAwsElastiCacheReplicationGroupId(v interface{}, k string) (ws []str
 	}
 	return
 }
+
+func validateAwsElastiCacheReplicationGroupAuthToken(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
Nitpick: We tend to keep our validation functions in `aws/validators.go` + tests in `aws/validators_test.go`. It would be nice to move it there too, it's not a blocker for this PR though.

Reply:
I'll move them!